e2e_tests/standard/test_base.py

```diff
@@ -175,19 +175,15 @@ def test_add_and_remove_tags(self, container: MetadataContainer):
 
 
 class TestFetchTable(BaseE2ETest):
-    def test_fetch_runs_table(self, environment):
+    def test_fetch_runs_by_tag(self, environment):
         tag1, tag2 = str(uuid.uuid4()), str(uuid.uuid4())
 
         with neptune.init_run(project=environment.project) as run:
             run["sys/tags"].add(tag1)
             run["sys/tags"].add(tag2)
-            run["value"] = 12
-            run.sync()
```

Comment:
Why have you removed it?

Reply:
Because `stop()` is called automatically when we leave context manager block.

Reply:
What about testing value od an attribute?

Reply:
This is `test_fetch_runs_by_tag` I wanted to check fetching by id only.
Testing of attribute values for runs is done in `test_fetch_runs_table`.

I think it's good to minimalize complexity of tests and make them focused on one aspect only.